db/migrations/0010.add_fkc_course_usage_table.py

```diff
@@ -0,0 +1,14 @@
+#
+# file: migrations/0010.add_fkc_course_usage_table.py
+#
+from yoyo import step
+
+__depends__ = {'0009.add_meta_tables'}
+
+step('''
+        ALTER TABLE canvas_course_usage
+        ADD CONSTRAINT fk_canvas_course_usage_id
```

Comment:
So the way I've been naming constraints is to reference what primary key you're linking to, so I think I would have done `fk_course_id`? What do you think?

Reply:
That's what I did, but it showed an error saying the key already exists probably you used it for enrollment table. so I changed to this

Reply:
Hm, yeah, I wondered about that. I think that's fine this way for now, pretty clear what you mean, and if there's confusion, there's the constraint itself which tells you what's going on.